edge_computing/cnf-talm-for-cluster-upgrades.adoc

```diff
@@ -8,8 +8,9 @@ toc::[]
 
 You can use the {cgu-operator-first} to manage the software lifecycle of multiple clusters. {cgu-operator} uses {rh-rhacm-first} policies to perform changes on the target clusters.
 
-:Featurename: Using PolicyGenerator resources with {ztp}
-include::snippets/technology-preview.adoc[]
+Using {rh-rhacm} and `{policy-gen-cr}` CRs is the recommended approach for managing and deploying policies to managed clusters.
```

Comment:
Is `{policy-gen-cr}` an attribute? If so, it isn't rendering in the doc preview.
```suggestion
Using {rh-rhacm} and `{policy-gen-cr}` CRs is the recommended approach for managing policies and deploying them to managed clusters.
```

Reply:
Thanks for the review @lahinson 
I've fixed the attribute